index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div>Regiss M.</div>
```

Comment:
### OPTIONAL
- [ ] Kindly change div to h1 tag, the h1 tag would be better since this is a text 🤗

Reply:
- change div tag to h1 tag

Reply:
I've done it